### PR TITLE
Main: Make help descriptions more consistent + add autocomplete

### PR DESCRIPTION
**Pull request by 5K1PY**

### CODE REVIEW

File: pisek/__main__.py [hunk 1]
```diff
@@ -108,215 +108,215 @@ def add_argument_dataset(parser):
...
     config_subparsers.add_parser(
-        "update", help="Update config to newest version. (Replaces the config.)"
+        "update", help="update config to newest version (replaces the config)"
     )
 
     # ------------------------------- pisek visualize -------------------------------
 
     parser_visualize = subparsers.add_parser(
-        "visualize", help="Show solutions statistics and closeness to limit."
+        "visualize", help="show solutions statistics and closeness to limit"
```
Comment (SvizelPritula) [inline pisek/__main__.py:225]: This should probably use the singular form of `solution`.

```suggestion
        "visualize", help="show solution statistics and closeness to limit"
```

File: pisek/__main__.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def add_argument_description(parser):
             "--description",
             "-d",
             type=str,
-            help="Created the dataset with a description of DESCRIPTION.",
+            help="create the dataset with a description DESCRIPTION",
```
Comment (SvizelPritula) [inline pisek/__main__.py:85]: This should be either `a description of DESCRIPTION` or `the description DESCRIPTION`.

**Review by SvizelPritula — CHANGES_REQUESTED** [review]
I found two small issues.

We're somewhat inconsistent in how many articles we use in the help messages, but I don't think that's too big of an issue.

File: pisek/__main__.py (hunk 1 above)
Comment (5K1PY) [reply pisek/__main__.py:225]: What? It visualizes multiple solutions.

Comment (SvizelPritula) [reply pisek/__main__.py:225]: Train tracks also aren't called trains tracks, even though multiple trains run on them.

Comment (5K1PY) [reply pisek/__main__.py:225]: Right, I'm stupid. Should read next time.

**Pull request merged**